monitoring-and-logging.md

```diff
@@ -0,0 +1,142 @@
+- The OpenShift Logging Operator has a dependency on the [Elasticsearch Operator here](https://console-openshift-console.apps.nerc-ocp-infra.rc.fas.harvard.edu/k8s/ns/openshift-operators-redhat/operators.coreos.com~v1alpha1~Subscription/elasticsearch-operator). Whether you use Elasticsearch for storing logs or using Loki, you still need the Elasticsearch Operator installed for required dependent CustomResourceDefinitions.
+
+### Loki operator
+
+- The Loki Operator allows you to set up LokiStacks, AlertingRules, RecordingRules, and RulerConfigs based on your cluster logs for infrastructure, audit, and applications. See the [ Loki Operator here ](https://console-openshift-console.apps.nerc-ocp-infra.rc.fas.harvard.edu/k8s/ns/openshift-operators-redhat/operators.coreos.com~v1alpha1~ClusterServiceVersion/loki-operator.5.4.6)
+- Setting up a LokiStack allows you to configure the size of a cluster logging system that you desire in terms of storage and replicas. [ LokiStack here ](https://console-openshift-console.apps.nerc-ocp-infra.rc.fas.harvard.edu/k8s/ns/openshift-operators-redhat/loki.grafana.com~v1beta1~LokiStack/lokistack)
+- Setting up a LokiStack involves configuring persistent storage by storageClassName for Persistent Volume Claims. [ ocs-external-storagecluster-ceph-rbd storage class here ](https://console-openshift-console.apps.nerc-ocp-infra.rc.fas.harvard.edu/k8s/cluster/storageclasses/ocs-external-storagecluster-ceph-rbd)
+- Setting up a LokiStack involves configuring object storage by a secret named "thanos-object-storage" in the "openshift-operators-redhat" namespace containing the access_key_id, access_key_secret, bucketnames, and endpoint of the object storage.
+- The object storage for Loki is provided by OpenShift Data Foundations. See the [ openshift-operators-redhat-objectbucketclaim Object Bucket Claim here ](https://console-openshift-console.apps.nerc-ocp-infra.rc.fas.harvard.edu/k8s/ns/openshift-operators-redhat/objectbucket.io~v1alpha1~ObjectBucketClaim/openshift-operators-redhat-objectbucketclaim)
+
+### Grafana operator
+
+- The Grafana Operator allows you to set up Grafana instances, datasources, dashboards, and notification channels. See the [ Grafana Operator here ](https://console-openshift-console.apps.nerc-ocp-infra.rc.fas.harvard.edu/k8s/ns/grafana/operators.coreos.com~v1alpha1~ClusterServiceVersion/grafana-operator.v4.6.0)
+- We configure a logging-grafana Grafana instance. See the [ Logging Grafana instance here ](https://console-openshift-console.apps.nerc-ocp-infra.rc.fas.harvard.edu/k8s/ns/grafana/clusterserviceversions/grafana-operator.v4.6.0/integreatly.org~v1alpha1~Grafana/logging-grafana)
```

Comment:
404.  version 4.8.0 rather than 4.6.0

Reply:
Thanks @aabaris , I have updated the link.